packages/react-heartwood-components/src/components/BigCalendar/components/DragGrid/DragGrid.js

```diff
@@ -966,34 +965,11 @@ class DragGrid extends PureComponent<Props, State> {
 		const {
 			className,
 			children,
-			onScroll,
```

Comment:
@gcpantazis these props were not supposed to be dropped into the `...props` spread so it would not be passed to child components and I don't see them being deleted from props after. I can't promise there won't be side effects here.

Reply:
Ugh. IMO the `...props` spread pattern is dangerous, I get the convenience but it causes a lot of unintended consequences. The reverse is true too, a benign prop that we add higher up might echo into components and cause side-effects. I think it's better to lean into flow and be fairly explicit with our types.

Here though... I'll take a gander and see what impact the props spread will cause. If I see anything particularly concerning I'll fix.

Reply:
@liquidg3 I took a look at how this spread was applied in code, and found that the only inbound prop that was being spread was `style`. So I got rid of the spread, killed the dead vars, and just passed the `style` prop to the root div directly. LMK if you have any concerns with that 🙌